scripts/testBot/runAqaArgParse.py

```diff
@@ -65,6 +66,22 @@ def main():
     args = vars(parser.parse_args(raw_args))
     # All args are lists of strings
 
+    # Help was requested. The simplest way to handle this is as if it were an error
+    # because stdout is reserved for workflow commands and logs.
+    if args['help']:
+        # For some unknown reason, the first tilde in this help message should not be escaped,
+        # otherwise a syntax error occurs when passed to the GitHub Script.
+        help_msg = '''Run AQA GitHub Action Documentation
+`\\`\\`
+https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/RunAqa.md
+\\`\\`\\`
+Click the above link to view the documentation for the Run AQA GitHub Workflow'''
+        sys.stderr.write(help_msg)
+        print('::error ::Help command invoked')
+        exit(2)
+    # Remove help flag from the args because it is not a parameter for the job matrix.
+    del args['help']
```

Comment:
Just wonder is it necessary to del help arg? If it's defined, the program is exit now. If not, do we need to del it?

Reply:
If `--help` is not specified then it will be stored a value of False.

Since the args get converted to JSON for the runBuild job matrix to parse, it will read `help: false` and throw an error because `help` is not an array of values.